docs/app/utils.js

```diff
@@ -18,7 +18,7 @@ export const parentComponents = _.flow(
 /**
  * Get the Webpack Context for all doc site examples.
  */
-export const exampleContext = require.context('docs/app/Examples/', true, /\.js$/)
+export const exampleContext = require.context('docs/app/Examples/', true, /(\w+Example\w+|index)\.js$/)
```

Comment:
This now excludes non-renderable other files, like `common.js`.  It is used in a few places, but only for rendering the example view components themselves.

>P.S.  The `common.js` files actually need to go as we cannot truly import any files into the examples since we're in a browser.  That is for another PR.